### DISCUSSION TITLE
Self-hosted runner on ppc64el architecture?

### DISCUSSION THREAD

**Discussion by wyphan** [community/community#24921] [Actions]
<p>Hi,</p>
<p>Are there plans to support the <code>ppc64el</code>/<code>ppc64le</code> (POWER) architecture for self-hosted Linux-based GitHub Action runners? Considering that both Debian, Ubuntu, and RHEL/CentOS already support this architecture. At the time of writing, it seems that only <code>amd64</code>, <code>armhf</code>, and <code>aarch64</code> architectures are supported…</p>
<p>I know it’s kinda niche, but I’m developing software that is able to run on this architecture, so I feel that I badly need this feature…</p>

**Comment by konradpabjan** [comment, marked as answer]
<p>Highly unlikely. Our <a href="https://github.com/actions/runner" rel="noopener nofollow ugc">open-sourced runner</a> is written using dotnet core which does not support the POWER architecture. I was able to find this: <a href="https://github.com/dotnet/runtime/issues/10055" rel="noopener nofollow ugc">https://github.com/dotnet/runtime/issues/10055</a></p>
<p>Also relevant: <a href="https://github.com/actions/runner/issues/243" class="inline-onebox" rel="noopener nofollow ugc">Expanded platform support: Consider porting to go or typescript · Issue #243 · actions/runner · GitHub</a></p>

**Comment by wyphan** [comment]
<p>I see. I guess for now I need to use Travis CI for ppc64le, then.</p>

**Comment by junaruga** [comment]
<p>I love to see GitHub Actions to support ppc64el architecture.</p>
<p>But do GitHub Actions really need to run the self hosted runner on ppc64el architecture directly? Is it possible for GitHub Actions as a proxy server, to connect to the external remote servers hosted by IBM to return the result?</p>

**Comment by junaruga** [comment]
<blockquote>
<p>But do GitHub Actions really need to run the self hosted runner on ppc64el architecture directly? Is it possible for GitHub Actions as a proxy server, to connect to the external remote servers hosted by IBM to return the result?</p>
</blockquote>
<p>Ah sorry I misunderstood this topic was about GitHub hosted runner.</p>
